cypress.json

```diff
@@ -20,5 +20,8 @@
     "*openx.net"
   ],
   "fixturesFolder": false,
-  "retries": 2
+  "retries": {
+    "runMode": 2,
+    "openMode": 0
```

Comment:
you can set these separately for `cypress run` and `cypress open` and I find that it's better to have a quicker feedback loop when a test fails while debugging [source](https://docs.cypress.io/guides/guides/test-retries#Configure-Test-Retries)